urls.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  // Private and public URLs are shown in the Nullstone UI
+  // Typically, they are created through capabilities attached to the application
+  // If this module has URLs, add them here as list(string)
+  additional_private_urls = []
```

Comment:
These are defined in `outputs.tf`.

Reply:
I removed these from outputs.tf since they are used here.